src/usersyncroniser.ts

```diff
@@ -233,10 +233,10 @@ export class UserSyncroniser {
         let mxidExtra = "";
         if (webhookID) {
             // no need to escape as this mxid is only used to create an intent
-            mxidExtra = `_${new MatrixUser(`@${webhookID}`).localpart}`;
+            mxidExtra = `_${new MatrixUser(`@${discordUser.username}`).localpart}`;
```

Comment:
So what was wrong with webhookID, it's not clear to me?

Reply:
the problem was that the same webhook can have different names and we don't want username changes constantly. So we fixed that by appending the name to the ID. Only, somehow here, the webhook id instead of the name was appended, which caused problems

Reply:
Hm, okay. Can this be commented above?

Reply:
done